docs/index.md

```diff
@@ -0,0 +1,320 @@
+# lolex
+
+> JavaScript implementation of the timer APIs
+
+**Implements:**
+
+* [`setTimeout`](#settimeoutcallback-timeout-args)
+* [`clearTimeout`](#cleartimeoutid)
+* [`setImmediate`](#setimmediatecallback-args)
+* [`clearImmediate`](#clearimmediateid)
+* [`setInterval`](#setintervalcallback-timeout-args)
+* [`clearInterval`](#clearintervalid)
+* [`requestAnimationFrame`](#requestanimationframecallback)
+* [`cancelAnimationFrame`](#cancelanimationframeid)
+* and many more ...
+
+along with a clock instance that controls the flow of time.
+**lolex** also provides a [`Date`](#date) implementation that gets its time from the clock.
+
+**lolex** can be used to simulate passing time in automated tests and other
+situations where you want the scheduling semantics, but don't want to actually
+wait (however, from version 2.0 **lolex** supports those of you who would like to wait too).
+
+**lolex** is extracted from [Sinon.JS](https://github.com/sinonjs/sinon.js).
+
+## Installation
+
+```shell
+npm install lolex --save-dev
+```
+
+You are always free to [build it yourself](https://github.com/sinonjs/lolex/blob/53ea4d9b9e5bcff53cc7c9755dc9aa340368cf1c/package.json#L22), of course.
```

Comment:
Why does this link to a specific commit hash?

Reply:
Just copied this, I'll change it to `master`

Reply:
Ahh, maybe the intention here was to not have to change this link if the `package.json` changes.